### ISSUE TITLE
Project 1

### ISSUE THREAD

**Issue by CKKKKKKKKK** [CKKKKKKKKK/Machine-Learning#1]
Data Splitting

- [x] Split the original ‘train.csv’ into ‘train.csv’, ‘valid.csv’ and ‘test.csv’
with the ratio of 0.8 : 0.1 : 0.1, respectively.

Data Preprocessing

- [x] Convert labels into to two classes: low (0, 1) and high (2, 3)

- [x] Data normalization (i.e., scaling values of attributes to
the same level, e.g., [0, 1])

- [x] For Naive Bayes, discretize continuous attributes into intervals

- [x] For Naive Bayes, split large number into ranges

Model Implementation

- [x] Implement Naïve Bayes

- [x] Implement Logistic Regression

Empirical Study

- [ ] Compare the three methods with respect to the classification
accuracy on the training set and the test set separately:
 Naive Bayesian
 SVM
 Logistic regression
Report the comparison of accuracy using figures and tables. 

- [ ] Report the time required by each of the methods, excluding the
time needed for loading and processing data. This may be done
using the time module. The result can be shown in a table.

- [ ] Provide a description for each figure or table. Analyze the
difference, pros and cons of the three methods.
